CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Fabric8-Launcher Backend Contributing Guide
+
+If you're looking for a development guide for the Fabric8-Launcher Backend, first read [the readme](./README.md).
+
+## First Steps
+
+### Prerequisites
+
+* Java 8+
+* Maven 3.5.3+
+* Git
+
+### GitHub Workflow
+
+To contribute to the Fabric8-Launcher Backend:
+
+1. Fork this repository to your own Git.
+2. Clone your fork.
+3. Commit/Push your work on topic branches
+4. Make pull requests.
+
+Here are 3 simple rules we follow:
+- Use a fork of our repository with a topic branch to contribute.
+- Redact a beautiful commit/pull-request message following our [guide](./COMMIT_MSG.md).
```

Comment:
I'd use "nice" instead of "beautiful", it sounds very lyrical/poetical to me now ;)